react/index.js

```diff
@@ -1,33 +1,17 @@
 import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
-import { injectIntl, intlShape } from 'react-intl'
-import hoistNonReactStatics from 'hoist-non-react-statics'
-import Modal from './components/Modal'
 import TopMenu from './components/TopMenu'
-import { Alert } from 'vtex.styleguide'
 import { ExtensionPoint, withRuntimeContext } from 'render'
 
-import {
-  orderFormConsumer,
-  contextPropTypes,
-} from 'vtex.store/OrderFormContext'
-
 import './global.css'
 
 class Header extends Component {
-  state = {
-    showMenuPopup: false,
-    leanMode: true,
-  }
-
   static propTypes = {
```

Comment:
It would be nice if you add some docs to these props, just like we have on the [minicart](https://github.com/vtex-apps/minicart/blob/master/react/propTypes.js) for example. :smiley: